docs/articles/expensify-classic/connect-credit-cards/company-cards/Troubleshooting.md

```diff
@@ -3,97 +3,156 @@ title: Troubleshooting
-5. **Address Gaps**: If there is a significant gap in transactions or if transactions are still missing, contact Expensify's Concierge or your Account Manager. They can initiate a historical data update on your card feed to ensure all transactions are properly imported.
+1. Go to **Settings** > **Domain** > _[Domain Name]_ > **Company Cards**.
+2. Click **Import Card/Bank** and follow the prompts.
 
-Following these steps should help you identify and resolve any issues with missing card transactions in Expensify.
+{% include info.html %}
+Only Domain Admins can connect and assign company cards in Expensify. If you're not a Domain Admin and want to connect your own credit card, follow the steps [here](https://help.expensify.com/articles/expensify-classic/connect-credit-cards/Personal-Credit-Cards) to connect it as a personal card.
+{% include end-info.html %}
 
-## Known issues importing transactions
-The first step should always be to "Update" your card, either from Settings > Your Account > Credit Card Import or Settings > Domain > [Domain Name] > Company Cards for centrally managed cards. If a "Fix" or "Fix card" option appears, follow the steps to fix the connection. If this fails to import your missing transactions, there is a known issue whereby some transactions will not import for certain API-based company card connections. So far this has been reported on American Express, Chase and Wells Fargo. This can be temporarily resolved by creating the expenses manually instead:
+## Best practices for establishing the initial card connection
+To ensure a successful initial card connection in Expensify, follow these best practices:
 
-- [Manually add the expenses](https://help.expensify.com/articles/expensify-classic/expenses/expenses/Add-an-expense)
-- [Upload the expenses via CSV](https://help.expensify.com/articles/expensify-classic/bank-accounts-and-credit-cards/company-cards/CSV-Import)
+- **Import in the Correct Location**: For company cards, navigate to **Settings** > **Domains** > _[Domain Name]_ > **Company Cards** > **Import Card** to establish the connection. For personal or individual card accounts, refer to the instructions [here](https://help.expensify.com/articles/expensify-classic/connect-credit-cards/Personal-Credit-Cards). 
+- **Select the Appropriate Bank Connection**: Ensure you’re selecting the appropriate bank connection for your cards.
+- **Use Master or Parent Administrative Credentials**: For company cards, always use the master administrative credentials to import the entire set of cards.
+- **Disable Two-Factor Authentication (2FA)**: Expensify cannot bypass bank-imposed 2FA requirements. To maintain a stable connection, temporarily disable 2FA on your bank account before attempting to connect.
```

Comment:
I won't block on this, but it seems like a strange suggestion. Why can't we maintain a stable connection if their bank has 2FA? That seems like something we should fix.